specification/managementgroups/resource-manager/Microsoft.Management/preview/2024-01-01-preview/management.json

```diff
@@ -0,0 +1,2383 @@
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The fully qualified ID for the management group.  For example, /providers/Microsoft.Management/managementGroups/0000000-0000-0000-0000-000000000000",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of the resource.  For example, Microsoft.Management/managementGroups",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the management group. For example, 00000000-0000-0000-0000-000000000000",
+          "readOnly": true
+        },
+        "properties": {
+          "title": "Properties",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ManagementGroupProperties"
+        }
+      }
+    },
+    "ManagementGroupProperties": {
+      "description": "The generic properties of a management group.",
+      "type": "object",
+      "properties": {
+        "tenantId": {
+          "type": "string",
+          "description": "The AAD Tenant ID associated with the management group. For example, 00000000-0000-0000-0000-000000000000"
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The friendly name of the management group."
+        },
+        "details": {
+          "title": "Details",
+          "$ref": "#/definitions/ManagementGroupDetails"
+        },
+        "children": {
+          "description": "The list of children.",
+          "x-nullable": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagementGroupChildInfo"
+          }
+        }
+      }
+    },
+    "ManagementGroupDetails": {
+      "description": "The details of a management group.",
+      "type": "object",
+      "properties": {
+        "version": {
+          "type": "number",
+          "format": "int32",
+          "description": "The version number of the object."
+        },
+        "updatedTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The date and time when this object was last updated."
+        },
+        "updatedBy": {
+          "type": "string",
+          "description": "The identity of the principal or process that updated the object."
+        },
+        "parent": {
+          "title": "Parent",
+          "$ref": "#/definitions/ParentGroupInfo"
+        },
+        "path": {
+          "description": "The path from the root to the current group.",
+          "x-nullable": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagementGroupPathElement"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "managementGroupAncestors": {
+          "description": "The ancestors of the management group.",
```

Comment:
>The ancestors of the management group. [](http://example.com/codeflow?start=26&length=38)

Is this an array of resourceIds of the parent MGs? Please add more information to the description.